xml_inscriptions/transcribed/CA.Malibu.JPGM.L.75.AD.38.xml

```diff
@@ -181,11 +182,22 @@
             <p/>
          </div>
          <div type="commentary">
-            <p/>
+            <p>
+               <lb n="1"/>form: perforated orbiculus, located at the top of the stamp extending through two lines of text and into the center of the stamp
+               <lb n="2"/>BDRCO 531
+               
+               <lb n="3"/>     "broken on all sides resuling in damage to outer edges of stamp and loss of line beginning"
+               
+               <lb n="4"/>dating: consular
```

Comment:
Couldn't this go in the dating criteria (`@evidence`)? Again, there's an existing place in the template for this.

Reply:
Add to dating criteria but leave also in commentary. There is never a reason NOT to clarify / specify / justify information in the commentary, which can include miscellaneous information.